pandas/io/excel.py

```diff
@@ -138,16 +139,18 @@ def read_excel(io, sheetname=0, header=0, skiprows=None, skip_footer=0,
           column ranges (e.g. "A:E" or "A,C,E:F")
     squeeze : boolean, default False
         If the parsed data only contains one column then return a Series
-    na_values : list-like, default None
-        List of additional strings to recognize as NA/NaN
+    na_values : str or list-like or dict, default None
+        Additional strings to recognize as NA/NaN. If dict passed, specific
+        per-column NA values. By default the following values are interpreted
+        as NaN: `'""" + "'`, `'".join(sorted(_NA_VALUES)) + """'`.
```

Comment:
I am afraid this does not work with docstrings. You will have to hardcode them like it is done in the `read_csv` docstring